software/spmd/amoadd_test/Makefile

```diff
@@ -0,0 +1,16 @@
+export BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)
+
+bsg_tiles_X = 4
+bsg_tiles_Y = 4
```

Comment:
Can we make it so that it uses the full manycore array to test? You would just need to comment out these lines.

Reply:
Sure! Reflected in the latest commit.